docs/changes.xml

```diff
@@ -5,6 +5,13 @@
 <change_log title="unit">
 
 
+<change type="bugfix">
+<para>
+fixed contents of SCRIPT_NAME in Ruby to always be an empty string.
+</para>
+</change>
+
+
 <changes apply="unit-php
```

Comment:
This should be placed into 1.28.0 changes.

Reply:
Rebased to master, where 1.28.0 changes exist, and moved there.